src/domain/communication/messaging/messaging.entity.ts

```diff
@@ -0,0 +1,24 @@
+import { Entity, OneToMany, OneToOne } from 'typeorm';
+import { AuthorizableEntity } from '@domain/common/entity/authorizable-entity';
+import { Conversation } from '../conversation/conversation.entity';
+import { IMessaging } from './messaging.interface';
+import { Platform } from '@platform/platform/platform.entity';
+
+@Entity('messaging')
+export class Messaging
+  extends AuthorizableEntity
+  implements IMessaging
+{
+  @OneToMany(
+    () => Conversation,
+    conversation => conversation.messaging,
+    {
+      eager: false,
+      cascade: true,
+    }
+  )
+  conversations!: Conversation[];
+
+  @OneToOne(() => Platform, platform => platform.messaging)
```

Comment:
Typically we have the one-one relationship set at the parent only. Otherwise get problems with circular dependencies in the code.